pulsar-broker/src/main/java/org/apache/pulsar/broker/delayed/DelayedDeliveryTrackerDisable.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.broker.delayed;
+
+import org.apache.bookkeeper.mledger.impl.PositionImpl;
+
+import java.util.NavigableSet;
+import java.util.concurrent.CompletableFuture;
+
+class DelayedDeliveryTrackerDisable implements DelayedDeliveryTracker {
```

Comment:
Why do we need this class

Reply:
This is just for fallback, to handle InMemoryDelayedDelieveryTracker initialize fail(even it should never fail). I can move the class to `DelayedDeliveryTracker` as an anonymous class